editor.planx.uk/src/pages/FlowEditor/components/Flow/components/Tag.tsx

```diff
@@ -1,18 +1,19 @@
 import Box from "@mui/material/Box";
 import { Palette, useTheme } from "@mui/material/styles";
-import { NodeTag } from "@opensystemslab/planx-core/types";
+import { NodeTag, Role } from "@opensystemslab/planx-core/types";
 import { useStore } from "pages/FlowEditor/lib/store";
 import React from "react";
 import { getContrastTextColor } from "styleUtils";
 import { FONT_WEIGHT_SEMI_BOLD } from "theme";
 
 export const TAG_DISPLAY_VALUES: Record<
   NodeTag,
-  { color: keyof Palette["nodeTag"]; displayName: string }
+  { color: keyof Palette["nodeTag"]; displayName: string; isEditable?: Role }
 > = {
   placeholder: {
     color: "blocking",
     displayName: "Placeholder",
+    isEditable: "platformAdmin", // if new roles are added, we should update the canEdit() in ComponentTagSelect.tsx
```

Comment:
I'd expect a variable starting with `is` to be a boolean. This would be better with a different name (and maybe an array of roles?) if we want to just list roles here. 

It seems like a simpler option would be a boolean. It looks like you hit a test issue when trying this when I check out [the commit history](https://github.com/theopensystemslab/planx-new/pull/3959/commits/55c3236b1fbcec69fc71ee304d675ccbeffa1c95). Did you manage to work out what the issue here was? It might be a relatively simple case of just mocking the useStore import in these tests.

Reply:
Ah fair enough, this makes sense! In which case, I think that updating variable name and maybe making this a `Role[]` is the way to go here 👍